jwala-services/src/main/java/com/cerner/jwala/service/jvm/impl/JvmServiceImpl.java

```diff
@@ -247,6 +247,12 @@ public Jvm updateJvm(final UpdateJvmRequest updateJvmRequest, final boolean upda
 
         updateJvmRequest.validate();
 
+        Jvm originalJvm = getJvm(updateJvmRequest.getId());
+        if (!originalJvm.getJvmName().equalsIgnoreCase(updateJvmRequest.getNewJvmName()) &&
+                !originalJvm.getState().toStateLabel().equals(JvmState.JVM_NEW.toStateLabel())) {
+            throw new JvmServiceException("Jvm " + originalJvm.getJvmName() + " is not in state " + JvmState.JVM_NEW.toStateLabel() + ", please do not rename Jvm");

```

Comment:
We need to change message, something like
"Jvm {} is in {} state, can only rename new jvm's,  delete then create new", originalJvm.getJvmName(), JvmState.JVM_NEW.toStateLabel()